Assets/Scripts/Control&Input/Peer2PeerTransport.cs

```diff
@@ -216,6 +251,23 @@ public override void OnClientDisconnect()
             ResetState();
     }
 
+    private void OnPlayerLeft(PlayerLeftMessage message)
+    {
+        if (!players.TryGetValue(message.id, out var playerDetails))
+        {
+            Debug.LogError($"Received leave message for invalid player {message.id}");
```

Comment:
We should probably consider logging some of these to players instead of just debug.log, we can probably set up a separate issue for that though

Reply:
Yeah, don't we already have an issue for killfeed?